xapian-letor/api/featurevector_internal.cc

```diff
@@ -117,3 +117,31 @@ FeatureVector::Internal::get_nonzero_num(){
     //cout << "nonzero: "<< nonzero <<endl;
     return nonzero;
 }
+
+void
+FeatureVector::Internal::normalise() {
+
+    int num_features = 19;
```

Comment:
I plan to get rid of these hard-coded values of '19's in the next `Features` PR. Can be merged for now.